README.md

```diff
@@ -90,6 +90,33 @@ but from here onwards you can use the `pganalyze` user instead.
 The collector will automatically use the helper methods
 if they exist in the `pganalyze` schema - otherwise data will be fetched directly.
 
+If you use `enable_log_explain`:
+
+```
+CREATE OR REPLACE FUNCTION pganalyze.explain(query text, params text) RETURNS text AS
+$$
+DECLARE
+  result text;
+BEGIN
+  -- this is a check to minimize collector access to your data: it ensures that the
+  -- collector cannot piggyback other queries that could exfiltrate data
+  IF query LIKE '%;%' OR params LIKE '%;%' THEN
+    RAISE EXCEPTION 'cannot run EXPLAIN when query or parameters contain semicolon';
```

Comment:
We may want to special-case this error in the UI like we are special-casing permissions issues.